DEVELOPMENT.md

```diff
@@ -25,6 +25,45 @@ Dependencies may potentiatially be out of date. In order to identify any out of
 
 Run `flutter pub get` to update the package dependencies to the new versions. `pubspec.lock` will be updated to save the new concrete package versions
 
+## Adding Plugin Support For New Platforms
+
+Adding support for different platforms may be wanted or necessary, so below is an example of adding support for iOS to the Tozny Flutter plugin. Run this command at the top level directory of the repository. Tozny plugin has partial support for iOS, so the below command needs not be run.  
+
+`flutter create --template=plugin --platforms=ios --ios-language=swift --org=com.tozny .`
+
+### `flutter create`
+
+The `flutter create` commmand defaults to using the application template, so `--template` must be set to `plugin`. The platform the plugin code is autogenerated for is defined using the `--platforms` option. Some platforms may have different language options (e.g. `swift` and `objc` for the `ios` platform), so specifying the language may be required. The `--org` option will be used for package naming in Java and as a prefix in iOS's bundle identifier. 
+
+### Possible Issues
+
+The repository in which the platform support is generated for needs to have a Dart package compliant name. Specifically, package names should be lowercase with words separated by underscores. 
+
+The `flutter create` command will complain if the repository name does not adhere to this convention, so make sure to rename the repository's name if necessary. 
+
+At the time of this writing the Tozny plugin's repository is named such that it will encounter this problem. 
```

Comment:
I am going to rename this repository so that it is compliant. This will require existing clones of the repo to update their upstream URL. To do that in the directory run`git remote set-url origin git@github.com:tozny/flutter_plugin`